board/miyoo/main/options.cfg

```diff
@@ -1,5 +1,4 @@
 MODULES_CUSTOM=0 
-FAT_CHECK=1 
```

Comment:
btrfs check(8)? How we can check data integrity?

Reply:
this is the beauty of btrfs, you dont have to manually check, as it is done automatically